test/gtest/ucp/test_ucp_tag_match.cc

```diff
@@ -515,8 +514,9 @@ UCS_TEST_P(test_ucp_tag_match, rndv_truncated, "RNDV_THRESH=1048576") {
     /* receiver - get the RTS and put it into unexpected */
     short_progress_loop();
 
-    /* receiver - issue a receive request, match it with the RTS and perform rndv get */
-    status = recv_b(&recvbuf[0], (recvbuf.size())/2, DATATYPE, 0x1337, 0xffff, &info);
+    /* receiver - issue a receive request with zero length,
+     * no assertions should occur */
+    status = recv_b(NULL, 0, DATATYPE, 0x1337, 0xffff, &info);
```

Comment:
can we keep the old test, so we eventually test both with zero and non-zero receive length?